appdatabase/migrations/sql/0001_app.up.sql

```diff
@@ -12,7 +12,9 @@ storage TEXT,
 pubkey BLOB,
 path TEXT,
 name TEXT,
-color TEXT
+color TEXT,
+created_at DATETIME,
```

Comment:
```suggestion
created_at DATETIME NOT NULL,
```

Reply:
This column can benefit from having an index.

Reply:
I can't imagine the volume would make much of a difference, but I'm on it.